core-bundle/src/Resources/contao/drivers/DC_Folder.php

```diff
@@ -1664,7 +1672,7 @@ public function editAll()
 			{
 				try
 				{
-					$this->denyAccessUnlessGranted(ContaoCorePermissions::DC_ACTION_EDIT, new DataContainerSubject($this->strTable, $id));
+					$this->denyAccessUnlessGranted(ContaoCorePermissions::DC_PREFIX . $this->strTable, new UpdateAction($this->strTable, array('id' => $this->intId)));
```

Comment:
should probably be `$id` not `$this->intId`

Reply:
Fixed in 3b6d2b4e48cbebb7c4ad5f4b8166df8348bfa209